examples/train_model.py

```diff
@@ -172,9 +177,11 @@ def main(parser):
                         time_left = min(time_left, other_time_left)
                     else:
                         time_left = other_time_left
-                if time_left is not None:
+                if time_left is not None and max_exs != 0:
                     logs.append('time_left:{}s'.format(math.floor(time_left)))
-
+                if opt['num_epochs'] > 0:
+                    display_epochs = int(total_exs / len(world)) if sampling_data else num_epochs
```

Comment:
thought we were trying to get rid of using len ?

Reply:
we've always had in in train_model.py on line 125